### PR TITLE
[IMP]10709 add module membership_account_balance

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: membership_account_balance/README.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Membership Management - Member Account Balance
```
Comment (elicoidal) [inline membership_account_balance/README.rst:6]: To be proposed to the OCA directly

Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: membership_account_balance/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline membership_account_balance/__init__.py:1]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: membership_account_balance/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{'name': 'Membership Management - Member Account Balance',
+ 'version': '0.1',
```
Comment (elicoidal) [inline membership_account_balance/__openerp__.py]: 8.0.1.0.0

File: membership_account_balance/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ 'category': '',
+ 'depends': ['account', 'membership', 'base_setup'],
+ 'author': 'Elico Corp',
+ 'license': 'AGPL-3',
+ 'website': 'https://www.elico-corp.com',
+ 'description': """
```
Comment (elicoidal) [inline membership_account_balance/__openerp__.py]: remove

File: membership_account_balance/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Total membership is calculated in the customer.
+
+Contributors
+------------
+
+* Alex Duan <alex.duan@elico-corp.com>
+* Siyuan Gu: <gu.siyuan@elico-corp.com>
+""",
+ 'images': [],
```
Comment (elicoidal) [inline membership_account_balance/__openerp__.py]: remove empty keys

File: membership_account_balance/README.rst (hunk 1 above)
Comment (Rona111) [inline membership_account_balance/README.rst:6]: @elicoidal  Do you know which project_repo does these module push to? I shoud add the project_repo in the README.rst.

Comment (elicoidal) [inline membership_account_balance/README.rst:6]: Point of Sales
Eric CAUDAL --
_Eric Caudal_ /[Founder and CEO]/
Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 
8017/27/37 (Office)
_Elico Shanghai_ (Hong Kong/Shenzhen/Singapore) 
http://www.elico-corp.com /_Odoo Gold Partner, best Odoo Partner 2014 
for APAC_/

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

File: membership_account_balance/README.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+Credits
+=======
+
+Contributors
+------------
+
+* Alex Duan <alex.duan@elico-corp.com>
+* Gu SiYuan <gu.siyuan@elico-corp.com>
+
+Maintainer
```
Comment (elicoidal) [inline membership_account_balance/README.rst:51]: Improve this information. Is it to be aimed at OCA?

**Comment by elicoidal** [comment]
@Rona111 please make a PR for the OCA directly

File: membership_account_balance/README.rst (hunk 6 above)
Comment (Rona111) [inline membership_account_balance/README.rst:51]: @elicoidal  yes,this module should be push to OCA.

**Comment by Rona111** [comment]
@elicoidal  can I push the module to OCA after I fixed all the  odoo module? I will push the module base the file modules_per_category_v2.

**Comment by Rona111** [comment]
@elicoidal @seb-elico @siyuancn @duanyp1991 Improved the readme.

**Comment by elicoidal** [comment]
@Rona111 I would push it asap to OCA (more important than our repository actually)

**Comment by Rona111** [comment]
@elicoidal  ok,Tks.

**Comment by noahzaozao** [comment]
@Elico-Corp/odoo-reviewers , please review this PR, thanks.

**Comment by seb-elico** [comment]
@noahzaozao As mentioned by @elicoidal we should make a PR on OCA/pos. But since we have elico-addons module(s) depending on this one right now, let's merge it here and see when/how we can move it to OCA/pos in the future.

LGTM (since it's v8, I don't bother about OCA guidelines, etc.)

**Comment by seb-elico** [comment]
@Elico-Corp/odoo-reviewers please check it

**Pull request merged**